spec/controllers/coronavirus_form/check_answers_controller_spec.rb

```diff
@@ -28,13 +29,45 @@
 
     it "saves the form response to the database" do
       session["attribute"] = "key"
+      session["medical_equipment"] = I18n.t("coronavirus_form.questions.medical_equipment.options.option_yes.label")
+      session["contact_details"] = { "contact_name" => "Harry Potter" }
       post :submit
 
       expect(FormResponse.first.form_response).to eq(
-        [%w[attribute key], %w[reference_number abc]],
+        "attribute" => "key",
+        "contact_details" => { "contact_name" => "Harry Potter" },
+        "medical_equipment" => "Yes",
+        "reference_number" => "abc",
```

Comment:
I wonder if this test could use `valid_data` as well? So as below, use `session.merge!(valid_data)`  in the set-up, in in the expectation use:

`expect(FormResponse.first.form_response).to eq(valid_data.merge(reference_number: "abc").deep_stringify_keys.merge`

Reply:
This test is probably not necessary, since it's a duplicate of the other test - I'll remove it